.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,40 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
```

Comment:
Did you look at alternatives to on -> push-> development? I saw some examples using conditionals and such things to simulate on merge / on pull request completion. By the way, I also am going with on -> push -> development.

Reply:
No, I didn't it did what was required, so I went with it.